docs/theme/extra/templates/navbar_links.html

```diff
@@ -5,7 +5,8 @@
 		Modules <span class="caret"></span>
 	</a>
 	<ul class="dropdown-menu" id="modules-menu">
-		@for tup in (("Gnome-module.html","GNOME"), \
+		@for tup in (("Fs-module.html","Filesystem"), \
+		       ("Gnome-module.html","GNOME"), \
```

Comment:
Indentation seems wrong

Reply:
Sorry about the small nitpicking, looks all good to me otherwise. Maybe also add the other missing modules in that list while you're here, it's totally outdated.